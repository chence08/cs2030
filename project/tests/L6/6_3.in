2 2 10
0.000000
0.313141
0.416894
1.116416
1.130640
1.284813
1.297472
2.775189
5.368209
5.665056
0.313263
2.645188
2.701273
0.263761
1.481332
0.415031
0.214836
3.512654
0.209277