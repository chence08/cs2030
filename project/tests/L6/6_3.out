0.000 1 arrives
0.000 1 serves by 1
0.313 2 arrives
0.313 2 serves by 2
0.313 1 done serving by 1
0.417 3 arrives
0.417 3 serves by 1
1.116 4 arrives
1.116 4 waits at 1
1.131 5 arrives
1.131 5 waits at 1
1.285 6 arrives
1.285 6 waits at 2
1.297 7 arrives
1.297 7 waits at 2
2.775 8 arrives
2.775 8 leaves
2.958 2 done serving by 2
2.958 6 serves by 2
3.118 3 done serving by 1
3.118 4 serves by 1
3.222 6 done serving by 2
3.222 7 serves by 2
3.637 7 done serving by 2
4.599 4 done serving by 1
4.599 5 serves by 1
4.814 5 done serving by 1
5.368 9 arrives
5.368 9 serves by 1
5.665 10 arrives
5.665 10 serves by 2
5.874 10 done serving by 2
8.881 9 done serving by 1
[1.008 9 1]